src/Transforms/Image.php

```diff
@@ -0,0 +1,64 @@
+<?php namespace Leean\Acf\Transforms;
+
+/**
+ * Class Image.
+ *
+ * @package Leean\Acf\Transforms
+ */
+class Image
+{
+	/**
+	 * Apply the transforms
+	 *
+	 * @param array $field The field.
+	 * @return array
+	 */
+	public static function apply( $field ) {
+		if ( ! ( 'image' === $field['type'] && 'id' === $field['return_format']) ) {
+			return $field['value'];
+		}
+
+		self::transform_image_fields( $field );
+
+		return $field['value'];
+	}
+
+	/**
+	 * Do the image size transform.
+	 *
+	 * @param string $field			Field
+	 * @return array
+	 */
+	public static function transform_image_fields( &$field ) {
```

Comment:
I think we should avoid passing a param by reference since it allow to overwrite the passed function I think the function should return the data as a new array and assign that on the function call in my opinion makes things more easier to understand at a first glance.